vendor/github.com/sn-srdjan/cx-tracker-cli/src/cli/cxtracker/tracker_provider.go

```diff
@@ -0,0 +1,91 @@
+package cxtracker
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	"github.com/sn-srdjan/cx-tracker-cli/src/api"
+)
+
+// Provider - provider object
+type Provider struct {
+	ServiceURL string
+	apiClient  *api.Client
+}
+
+// CXApplicationConfig holds config describing CX Application instance
+type CXApplicationConfig struct {
+	//TODO confirm should config contain genesis hash and work based on it (alternative needing to use config file content hash), and
```

Comment:
This is another important topic.
One part is trivial - to confirm is genesis hash the same thing as genesis signature. I saw both referenced in the project, think they are the same but would like to double check that info. Signature is mentioned in steps needed to setup the chain and I know hash was mentioned before and I also se references in the code so...

Another is not so trivial. We talked about having CX Tracker store CX Application configs.
There is a concept of hash representing content of the config file. And there is concept of Genesis Hash (if previous discussion topic makes any sense).
Pros and Cons of using either of two hashes is detailed in code so let's discuss on how to proceed with this.

Reply:
First point: Yes, genesis hash is the same as the genesis signature.

Second point: As far as I understand, we only need a way to uniquely identify the CX application, i.e. we need anything that will be unique and that it will be attached to that CX application. If you already have access to the genesis hash, and this will be unique, I think it can be used to access information for that entry (like retrieving the config file).